jena-arq/src/main/java/org/apache/jena/riot/RDFLanguages.java

```diff
@@ -293,41 +294,27 @@ public static void register(Lang lang) {
         }
     }
 
-    private static boolean isMimeTypeRegistered(Lang lang) {
```

Comment:
Please leave this function and keep its usage. If it needs fixes,fix it. 

Having named operations for specific steps makes the code clearer. The optimizer will deal with efficiency.